modules/xfem/test/tests/single_var_constraint_2d/equal_value.i

```diff
@@ -0,0 +1,96 @@
+[GlobalParams]
+  order = FIRST
+  family = LAGRANGE
+[]
+
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 5
+  ny = 5
```

Comment:
This is a quasi-1D problem. Why even have any more elements than 1 in the y direction?